Install.md

```diff
@@ -87,19 +68,25 @@ This guide helps you build and install Snort.
     ```sh
     cd snort-2.9.8.3/
     ```
-    
-2. Run the configuration script.
+2. Prepare for automake, then autocreate makefile. 
     ```sh
-    ./configure --enable-sourcefire
+    aclocal
+    autoconf
+    autoheader
+    automake -a
+    ```
+3. Run the configuration script.
+    ```sh
+    ./configure --enable-sourcefire --enable-static --disable-shared
     ```
 
-3. Create Patch of Makefille      
+4. Create Patch of Makefille. This part is critical for the next steps. The defaults are blank. Please type in the answers manually.
```

Comment:
Did you input anything specific here? Without the patch, the build completes. If you didn't enter anything specific, lets remove this step (traditional snort install doesn't do this)

Reply:
Yes, I do not know the rationale behind this. Perhaps someone else could expand.